src/ansys/dpf/core/operator_specification.py

```diff
@@ -231,7 +232,7 @@ class Specification(SpecificationBase):
     'result file path container, used if no streams are set'
     """
 
-    def __init__(self, operator_name=None, specification=None, server=None):
+    def __init__(self, operator_name: Union[str, None]=None, specification: Union["Specification", None]=None, server: Union[server_module.BaseServer, None]=None):
```

Comment:
Again, you could use `Specification` directly by using `from typing import TYPE_CHECKING` and `if TYPE_CHECKING: import Specification`, you can see it being done in `GenericDataContainer.py`.
Also `from __future__ import annotations` as a first import

Reply:
I was afraid this could cause problems on runtime to our consumers, couldn't it?